GPU/Common/FramebufferCommon.cpp

```diff
@@ -532,15 +532,17 @@ void FramebufferManagerCommon::NotifyRenderFramebufferSwitched(VirtualFramebuffe
 				// to it. This broke stuff before, so now it only clears on the first use of an
 				// FBO in a frame. This means that some games won't be able to avoid the on-some-GPUs
 				// performance-crushing framebuffer reloads from RAM, but we'll have to live with that.
+
+				// Wait, can we even do this? Seems highly unsafe.. TODO: Remove
```

Comment:
Yeah, this has been here a while and only done on GLES (not sure if that flag can be set under Vulkan?), but I agree it seems unsafe...

-[Unknown]

Reply:
I had actually totally forgotten that we tried this optimization here. Apparently it doesn't have many ill effects since so many games work, but I still don't like it and we should probably only use it on a whitelist basis, if anything.

I don't think the GLES flag is set in Vulkan but not 100% sure heh.

Maybe it can be another part of the explanation for those cases where GL has been faster than Vulkan on some devices. But we don't seem to hear much about that anymore...